lazy_tensor_core/lazy_bench.py

```diff
@@ -366,6 +373,24 @@ def check_fuser(args):
     if args.fuser == 'fuser1':
         assert torch._C._llvm_enabled(), "Can't use fuser1 (nnc) without building torch with llvm."
 
+def run_tracing_execute_noops(test, lazy_benchmark):
+    ltm.set_noop_execution_mode(True)
```

Comment:
would it make sense to also set `torch.cuda.set_sync_debug_mode(2)` to make sure we aren't running any ops?

Reply:
oops missed this.

yea, maybe we should.  I wasn't sure about where to add that debug mode in the default path, but i'll try it.